packages/composite-checkout-wpcom/src/hooks/wpcom-store.ts

```diff
@@ -47,8 +48,10 @@ export function useWpcomStore(
 		action: WpcomStoreAction
 	): ManagedContactDetails {
 		switch ( action.type ) {
-			case 'UPDATE_CONTACT_DETAILS':
+			case 'UPDATE_CONTACT_DETAILS': {
```

Comment:
Just so you know: these curly braces are not necessary in a case statement unless you are creating variables that need their own block scope.